recipes/pydna_repeatfinder/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "pydna_repeatfinder" %}
+{% set version = "0.2.8" %}
+{% set pypiname = "pydnarepeatfinder" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ pypiname[0] }}/{{ pypiname }}/{{ pypiname }}-{{ version }}.tar.gz"
+  sha256: "ceb25f38ae6baa74a943942bf59dea7e688c4535c7cf83de9ebd26ce18f74a03"
+
+build:
+  number: 0
+  skip: True  # [py < 37]
```

Comment:
```suggestion
```
Currently the oldest Python that Bioconda builds for is 3.8, so this is not really needed.